tests/dotless.Core.Test/Unit/Parser/ColorTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace dotless.Core.Test.Specs.Functions
```

Comment:
The namespace here should be dotless.Core.Test.Unit.Parser, I think.

Reply:
Ah, I copied another file and forgot to fix this, good catch.